src/components/Contentful/HomePage/NewHomeLoansByCategoryGrid.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+	<section-with-background-classic
+		:background-content="background"
+		:vertical-padding="verticalPadding"
+	>
+		<template #content>
+			<kv-page-container>
+				<div>
+					<kiva-multi-category-grid
+						:contentful-loan-channels="contentfulLoanChannels"
+						:loan-display-settings="loanDisplaySettings"
+					/>
+				</div>
+			</kv-page-container>
+		</template>
+	</section-with-background-classic>
+</template>
+
+<script>
+import contentfulStylesMixin from '@/plugins/contentful-ui-setting-styles-mixin';
+import KivaMultiCategoryGrid from '@/components/Homepage/HomeExp/KivaMultiCategoryGrid';
+import SectionWithBackgroundClassic from '@/components/Contentful/SectionWithBackgroundClassic';
+import KvPageContainer from '~/@kiva/kv-components/vue/KvPageContainer';
+
+export default {
+	name: 'NewHomeLoansByCategoryGrid',
+	components: {
+		KivaMultiCategoryGrid,
+		KvPageContainer,
+		SectionWithBackgroundClassic,
+	},
+	mixins: [contentfulStylesMixin],
+	props: {
+		/**
```

Comment:
Not super important to change, but something to watch in the future, these jsdoc blocks all have an extra asterick. If you type `/**`, you'll get a correct jsdoc automatically in VS Code above a property/method. The format should be:

```
/**
 * Comment
 */
```